explorer/app/components/Citations/Citations.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Item, ItemProps } from "./Item";
+
+interface CitationsProps {
+  citations: ItemProps[];
+  align?: "right" | "left";
```

Comment:
Hey @BruceRodrigues now that we've got alignment "right", "left" here and alignment "left", "center" in the header component should we make an enum for these values?